docs/standard/threading/scheduling-threads.md

```diff
@@ -12,7 +12,107 @@ ms.assetid: 67e4a0eb-3095-4ea7-b20f-908faa476277
-  
+
+## Example 
+Here is an example of the execution of 9 threads across all 5 priority levels from the [ThreadPriority](https://learn.microsoft.com/en-us/dotnet/api/system.threading.thread.priority) enumeration where the last 5 are on highest priority level. Also we have callback support from the previous article that in this context demonstrates that the order of thread initialization and prioritization may not always be reflected in subsequent code nor process executions start order. Meaning, we see here the parallel nature of code execution and demonstation of assigned processor time slices by the operating system for every thread. This highlights the influence and control of the environment in which threads are running. With that said, we certainly see that the highest priority threads are indeed receive priority in execution. 
+
+The following code will output arbitrary results on each execution. But common sequence patters of priorities being ran could be observed after running multiple times and analyzing the outputs.
+
+```c#
+public class ThreadPriorityExample
+{
+    public class ThreadWithCallback
+    {
+        public ThreadWithCallback(Action<ThreadPriority> callback)
+        {
+            this.callback = callback;
+        }
+
+        public Action<ThreadPriority> callback;
+
+        public void Process()
+        {
+            Console.WriteLine($"Entered process in {Thread.CurrentThread.Priority} priority.  \t\t ThreadId: {Thread.CurrentThread.ManagedThreadId}.");
+            Thread.Sleep(1000);
+            Console.WriteLine($"Finished process in {Thread.CurrentThread.Priority} priority. \t\t ThreadId: {Thread.CurrentThread.ManagedThreadId}.");
+
+            if (callback != null)
+            {
+                callback(Thread.CurrentThread.Priority);
+            }
+        }
+    }
+
+    public void RunMultipleThreadsOnDifferentPriorities()
+    {
+        Thread t0 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t1 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t2 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t3 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t4 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t5 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t6 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t7 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
+        Thread t8 = new Thread(() => { new ThreadWithCallback(Callback).Process(); });
```

Comment:
There's a log of opportunity to refactor this so it's got less repeated code. Maybe store the threads in an array and loop through the creation, setting priority, and start code?